schema/workerschema/v1.json

```diff
@@ -254,6 +254,19 @@
           "response": {
             "$ref": "ClientWithSecret"
           }
+        },
+        "Revoke": {
+          "id": "dex.Client.Revoke",
+          "description": "Revoke all refresh tokens issues to the client for the authenticated user.",
+          "httpMethod": "DELETE",
+          "path": "client/refresh/{clientid}/revoke",
```

Comment:
Revoke just because we might add other actions later. Happy to take it out.

I believe you asked it to be moved from '/account' to the '/client' prefix (though github seems to have lost that comment).